include/nbl/builtin/hlsl/subgroup/arithmetic_portability.hlsl

```diff
@@ -5,9 +5,13 @@
 #define _NBL_BUILTIN_HLSL_SUBGROUP_ARITHMETIC_PORTABILITY_INCLUDED_
 
 #ifndef NBL_GL_KHR_shader_subgroup_arithmetic
-#include "nbl/builtin/hlsl/subgroup/basic_portability.hlsl"
+#include "nbl/builtin/hlsl/subgroup/basic.hlsl"
 #endif
 
+// [[vk::ext_extension("GL_KHR_shader_subgroup_arithmetic")]] REVIEW-519: Extensions don't seem to be needed?
+[[vk::ext_capability(/* GroupNonUniformArithmetic */ 63)]]
+void spirv_arithmetic_extcap(){}
```

Comment:
can you namespace these, good place would be `nbl::hlsl::spirv::impl`

Also why is the arithmetic cap here when it should be together with the SPIR-V  header?

Reply:
done?

Reply:
yeah the `impl` stuff is unreachable, needs to be redone, convo in another comment.